xo-update.sh

```diff
@@ -13,6 +13,8 @@ echo $output updates available
 if [[ $output -ne 0 ]]; then
   echo "Updating from source..."
   sudo git pull
+  sudo rm -rf dist
+  sudo rm -rf node_modules
```

Comment:
This can cause issues with plugins disappearing. The solution is to install them under /usr/local/lib/node_modules instead.